test/test_format.py

```diff
@@ -375,7 +375,7 @@ def test_makeboxes_outputform_text(
         ("Subsuperscript[a, p, q]", "a_p^q", None),
         (
             "Integrate[F[x],{x,a,g[b]}]",
-            "\\int_a^{g\\left[b\\right]} F\\left[x\\right] \uf74cx",
```

Comment:
Ideally this kind of thing would be in a separate PR if not separate commit. It's not strictly necessary but something that would be nice to keep in mind. Thanks.